### DISCUSSION TITLE
PermissionError: [Errno 13] Permission denied: '/workspace/data/user'

### DISCUSSION THREAD

**Discussion by gpnath82** [h2oai/h2o-llmstudio#312] [General]
Why is is still showing the error ?

RUN adduser --uid 1999 llmstudio
RUN mkdir /workspace
RUN chown llmstudio -R /workspace
USER llmstudio
WORKDIR /workspace

**Comment by pascal-pfeiffer** [comment]
closing as it is fixed now https://github.com/h2oai/h2o-llmstudio/issues/313
